### PR TITLE
UI fixes

### PR DESCRIPTION
**Pull request by manojava-gk**

### CODE REVIEW

File: cx-portal/src/components/pages/EdcConnector/edcConnectorTableColumns.tsx [hunk 1]
```diff
@@ -53,34 +43,41 @@ export const ConnectorTableColumns = (
       headerName: t('content.edcconnector.columns.location'),
       flex: 0.5,
       sortable: false,
+      align: 'center',
+      headerAlign: 'center',
+      renderCell: ({ row }: { row: any }) => (
+        <Box>
+          <Typography variant="body1" sx={{ fontSize: '16px' }}>
```
Comment (jjeroch) [inline cx-portal/src/components/pages/EdcConnector/edcConnectorTableColumns.tsx:50]: @manojava-gk logical question:
why do we need to add `fontSize: '16px'` if we already mention `variant="body1`

File: cx-portal/src/components/pages/EdcConnector/edcConnectorTableColumns.tsx [hunk 2]
```diff
@@ -53,34 +43,41 @@ export const ConnectorTableColumns = (
...
+        </Box>
+      ),
     },
     {
       field: 'type',
       headerName: t('content.edcconnector.columns.type'),
       flex: 1,
       sortable: false,
+      renderCell: ({ row }: { row: any }) => (
+        <Typography variant="body1" sx={{ fontSize: '16px' }}>
```
Comment (jjeroch) [inline cx-portal/src/components/pages/EdcConnector/edcConnectorTableColumns.tsx:62]: same comment as above

File: cx-portal/src/components/pages/EdcConnector/edcConnectorTableColumns.tsx [hunk 3]
```diff
@@ -53,34 +43,41 @@ export const ConnectorTableColumns = (
...
+          {row.type === 'COMPANY_CONNECTOR' ? 'Owned' : 'Managed'}
```
Comment (jjeroch) [inline cx-portal/src/components/pages/EdcConnector/edcConnectorTableColumns.tsx:63]: Suggesting to already enable translation with the locales files

**Review by jjeroch — CHANGES_REQUESTED** [review]

File: cx-portal/src/components/pages/EdcConnector/edcConnectorTableColumns.tsx (hunk 1 above)
Comment (manojava-gk) [reply cx-portal/src/components/pages/EdcConnector/edcConnectorTableColumns.tsx:50]: It was needed bcz ```body1``` has ```fontSize 18px```. But now you have merged Typography changes to dev, I can use other variants

File: cx-portal/src/components/pages/EdcConnector/edcConnectorTableColumns.tsx (hunk 3 above)
Comment (manojava-gk) [reply cx-portal/src/components/pages/EdcConnector/edcConnectorTableColumns.tsx:63]: will move this

File: cx-portal/src/components/pages/EdcConnector/edcConnectorTableColumns.tsx [hunk 4]
```diff
@@ -53,34 +43,41 @@ export const ConnectorTableColumns = (
...
-            ) : (
-              <LockIcon sx={{ color: '#b6b6b6' }} />
-            )}
-          </IconButton>
+        <Box>
+          {row.dapsRegistrationSuccessful ? (
+            <LockIcon sx={{ color: 'green' }} />
+          ) : (
+            <LockIcon sx={{ color: '#b6b6b6' }} />
+          )}
```
Comment (oyo) [inline cx-portal/src/components/pages/EdcConnector/edcConnectorTableColumns.tsx:80]: write this with less duplication as
`<LockIcon sx={{ color: row.dapsRegistrationSuccessful ? 'green' : '#b6b6b6' }} />`

File: cx-portal/src/components/pages/EdcConnector/edcConnectorTableColumns.tsx [hunk 5]
```diff
@@ -53,34 +43,41 @@ export const ConnectorTableColumns = (
       headerName: t('content.edcconnector.columns.location'),
       flex: 0.5,
       sortable: false,
+      align: 'center',
+      headerAlign: 'center',
+      renderCell: ({ row }: { row: any }) => (
+        <Box>
+          <Typography variant="body2">{row.location}</Typography>
+        </Box>
```
Comment (oyo) [inline cx-portal/src/components/pages/EdcConnector/edcConnectorTableColumns.tsx:51]: Wrapping this in a <Box> seems unnecessary - the next `renderCell` below goes without it

**Review by oyo — CHANGES_REQUESTED** [review]

**Review by oyo — APPROVED** [review]
looks good from my side now

**Review by jjeroch — APPROVED** [review]

**Pull request merged**